feeds_jsonpath_parser.info

```diff
@@ -3,5 +3,6 @@ description = Parse a JSON document using JSONPath.
 backdrop = 1.x
 type = module
 dependencies[] = feeds
+dependencies[] = libraries
```

Comment:
Let's move this to another PR, as it's not related to this issue.

Reply:
Done - well taken out for now.
I've not had much success having more than one PR on the go